data/software-tools/bandit.json

```diff
@@ -1,23 +1,14 @@
 {
-  "@context": {
-    "schema": "http://schema.org/",
-    "rs": "https://w3id.org/everse/rs#"
-  },
+  "@context": "https://w3id.org/everse/rs#",
   "@id": "https://w3id.org/everse/tools/bandit",
-  "@type": "schema:SoftwareApplication",
-  "schema:name": "bandit",
-  "schema:description": "Bandit is a tool designed to find common security issues in Python code.",
-  "schema:url": "https://github.com/PyCQA/bandit",
-  "schema:isAccessibleForFree": true,
-  "rs:hasQualityDimension": {
-    "@id": "rs:Sustainability",
-    "@type": "@id"
-  },
-  "rs:howToUse": ["CI/CD", "command-line"],
-  "rs:appliesToProgrammingLanguage": ["Python"],
-  "schema:license": {
-    "@id": "https://opensource.org/licenses/Apache-2.0",
-    "@type": "@id"
-  },
-  "schema:applicationCategory": { "@id": "rs:AnalysisCode", "@type": "@id"}
-}
\ No newline at end of file
+  "@type": "SoftwareApplication",
+  "name": "bandit",
+  "description": "Bandit is a tool designed to find common security issues in Python code.",
+  "url": "https://github.com/PyCQA/bandit",
+  "isAccessibleForFree": true,
+  "hasQualityDimension": "Sustainability",
+  "howToUse": [ "CI/CD", "command-line" ],
+  "appliesToProgrammingLanguage": [ "Python" ],
+  "license": "https://opensource.org/licenses/Apache-2.0",
+  "applicationCategory": "AnalysisCode"
```

Comment:
should now be `rs:AnalysisCode`
see https://github.com/EVERSE-ResearchSoftware/schemas/pull/43#issuecomment-3032866947